tests/src/Interop/PInvoke/Delegate/DelegateAsInterfaceTestNative.cpp

```diff
@@ -8,6 +8,7 @@ namespace
 {
     HRESULT InvokeDelegate(IDispatch* dele, VARIANT* pResult)
     {
+        VariantInit(pResult);
```

Comment:
Can you move this closer to the `dele->Invoke()` call? It is weird to initialize an out prior to anything in a function utilizing COM semantics.